Description: fix false pulseaudio dependency of libQt5MultimediaWidgets
 The assignment 'QT += multimedia-private' exports multimedia-private's
 pulseaudio dependencies. Fix this by using 'multimedia' as public
 dependency and 'multimedia-private' as private dependency.
Origin: upstream, https://code.qt.io/cgit/qt/qtmultimedia.git/commit/?id=7e64b859b6a663f4
Last-Update: 2020-03-14

--- a/src/multimediawidgets/multimediawidgets.pro
+++ b/src/multimediawidgets/multimediawidgets.pro
@@ -1,6 +1,7 @@
 # distinct from Qt Multimedia
 TARGET = QtMultimediaWidgets
-QT = core gui multimedia-private widgets-private
+QT = core gui multimedia widgets-private
+QT_PRIVATE += multimedia-private
 qtHaveModule(opengl): \
    QT_PRIVATE += opengl
 
